lib/objects.js

```diff
@@ -1245,5 +1257,43 @@ module.exports = {
 			},
 			native: {}
 		}
+	],
+	'settings': [
```

Comment:
Ja, nenne es ruhig dreamSettings, denn die scheinen ja andere Werte für fan_power und mop modus zu haben

Reply:
Weiss noch nicht wie schlau das ist. Wenn ich verschiedene geräte habe, möchte ich vll. Nicht unbedingt immer andere Einstellungen berücksichtigen müssen? Wäre jetzt mein Argument.

Reply:
Das ist ja auch nicht Sinn, aber es macht Sinn, alle States an einer Stelle zu haben. Und je nach Modell holst du dir dann die Objekte raus, die du benötigst. Beseer wäre noch, wir würden da einmal ganz aufräumen und Herstellerunabhängige States zu machen. Der jeweilige manager müsste das dann mappen. Würde aber bedeuten, es müsste ziemlich viel neu gemacht werden. Das machen wir dann vlt. mal besser in einem neuen Adapter, der nur vacuum heist und die komplette Logig für Staubsauger abbildet. Wie es das Gerät dann macht, bzw, ob es das überhauüt kann, wird dann pro Hersteller/Model geregelt

Reply:
Ja, hatte auch schon überlegt die Kernfunktionen für die miot spec zu abstrahieren, dass man nur noch "konfigurieren" muss. Wäre von meinem Adapter kein weiter weg. Müsste dann nur ableiten. Dir bisherigen sind ja auch drin gelistet.

Ich Versuche derzeit ja auch so nenn bissl zu Mappen, sofern es geht.

Neuer Adapter macht es wahrscheinlich einfacher. Oder halt ne neue Version, die man jetzt schon anfängt und umbaut. Darf ja ruhig altes breaken dann.